src/cafe/farm.py

```diff
@@ -48,17 +52,52 @@ def setTrees(self):
         
         return(numTrees)
         
-    def inheretTreeProperties(self, treeType):
-        loop = True
+    def inheretTreeProperties(self, treeType, treeAttributes):
+        
         """
         Based on the argument treeType in the initializer function, assign parameters for the respective
         life & production patterns of the trees on the cuerda.
         
         The following property assignments were developed from data collected from the co-op in 2014
     
         """
-        while loop:
-            if (treeType =='borbon'):
+        treeType = treeType.lower() # convert to lower case for easier parsing
+        
+        if treeAttributes:
+            keys = list(treeAttributes.keys())
+            altOrth = [treeAttributes[key]['altOrth'] for key in treeAttributes]
+            # tipos = keys + altOrth # all of the possible spellings for the tree types
+            
+            if treeType in keys:
+                treeDict = treeAttributes[treeType]
+                
+            elif treeType in altOrth:
+                keyPair = [(key, treeAttributes[key]['altOrth']) for key in treeAttributes]
+                _treeType = ''
+                for i,e in enumerate(keyPair):
+                    if treeType == e[1]:
+                        _treeType = e[0]
+                
+                treeDict = treeAttributes[_treeType]
+                
+            else:
+                raise AttributeError(
+                """
+                '%s' is not a recognized value (orthography) in the `treeAttributes` dict.
+                
+                """%(treeType))
+                
+            self.treeType = treeDict['treeType']
+            self.firstHarvest = treeDict['firstHarvest']
+            self.fullHarvest = treeDict['fullHarvest']
+            self.descentHarvest = treeDict['descentHarvest']
+            self.death = treeDict['death']
+            self.cuerdaHarvestCap = treeDict['cuerdaHarvestCap']
+        
+        # as soon as yaml imports are tested, this can be deleted:
+        # temporary stand-in for variable testing
+        else:
+            if (treeType =='borbon') or (treeType == 'borbón'):
```

Comment:
use the `str` library or something similar to standardize stripping/formatting instead of trying to catch this